src/js/utils/functions.js

```diff
@@ -0,0 +1,8 @@
+export function pick(o, fields) {
```

Comment:
Instead of setting this precedent what do you think about https://lodash.com/custom-builds ?

(very much out of scope of this PR but I'm curious to know what you think)